### PR TITLE
Update weekly_report.js to remove old auth token

### PR DESCRIPTION
**Pull request by zackkrida**
This script isn't being used but I think it'd be best to remove this old, deleted key.

### CODE REVIEW

**Review by sarayourfriend — APPROVED** [review]

File: automations/js/src/weekly_report.js [hunk 1]
```diff
@@ -20,6 +20,14 @@ if (!date) {
   )
 }
 
+// get auth key
+const key = process.argv?.[3]
```
Comment (stacimc) [inline automations/js/src/weekly_report.js:24]: Should this be `ghKey` as used below?

**Review by obulat — APPROVED** [review]

File: automations/js/src/weekly_report.js (hunk 1 above)
Comment (zackkrida) [inline automations/js/src/weekly_report.js:24]:
```suggestion
const ghKey = process.argv?.[3]
```

File: automations/js/src/weekly_report.js [hunk 2]
```diff
@@ -20,6 +20,14 @@ if (!date) {
   )
 }
 
+// get auth key
+const ghKey = process.argv?.[3]
+if (!key) {
```
Comment (zackkrida) [inline automations/js/src/weekly_report.js:25]:
```suggestion
if (!ghKey) {
```

File: automations/js/src/weekly_report.js (hunk 1 above)
Comment (zackkrida) [reply automations/js/src/weekly_report.js]: yes, thanks

**Pull request merged**